src/components/views/dialogs/spotlight/RoomResultDetails.tsx

```diff
@@ -17,12 +17,31 @@ limitations under the License.
 import React from "react";
 import { Room } from "matrix-js-sdk/src/matrix";
 
+import { _t } from "../../../../languageHandler";
 import { roomContextDetailsText, spaceContextDetailsText } from "../../../../utils/i18n-helpers";
 
-export const RoomResultDetails = ({ room }: { room: Room }) => {
-    const contextDetails = room.isSpaceRoom() ? spaceContextDetailsText(room) : roomContextDetailsText(room);
+interface Props {
+    id: string;
+    room: Room;
+}
+
+export const RoomResultDetails = ({ room, id }: Props) => {
+    let contextDetails: string;
+    let ariaLabel: string;
+    if (room.isSpaceRoom()) {
+        contextDetails = spaceContextDetailsText(room);
+        ariaLabel = _t("In space %(space)s.", {
```

Comment:
We should pretty much never be interpolating a translated string into another, it isn't language agnostic plus weblate doesn't show the contexts the string can occur in so it can result in poor contextual translations even in languages where it'd otherwise be okay. 

https://github.com/vector-im/element-web/blob/develop/docs/translating-dev.md#things-to-knowstyle-guides

> Avoid "translation in parts", i.e. concatenating translated strings or using translated strings in variable substitutions. Context is important for translations, and translating partial strings this way is simply not always possible.

Reply:
You are absolutely right! I didn’t check the source of spaceContextDetails and only ever saw the case for a single space, so I didn’t expect that this might occur. Thanks for reminding me, I’ve changed this now.

I’d also like your opinion on whether https://github.com/matrix-org/matrix-react-sdk/pull/9003/files#diff-4f723693a224616ad86bb469ab940e993ed1fc02a91fe002e95b2aad28bd0f6fR263-R278 should be changed accordingly as well, I tried to copy the logic that we’re already using for the room list, but I feel a bit uneasy with concatenating translated strings.

Reply:
@justjanne sorry that link isn't working very well, it just bounces me to https://github.com/matrix-org/matrix-react-sdk/pull/9003/files# - could you point at the section in another manner?

Reply:
How about https://github.com/matrix-org/matrix-react-sdk/blob/e4c0d875e6802df7d0bb80671ab30d8b07c1e13e/src/components/views/dialogs/spotlight/SpotlightDialog.tsx#L263-L277, does that work for you?